### PR TITLE
chore: typo fixes, proxy IP availability

### PR DESCRIPTION
**Pull request by seyhello**

### CODE REVIEW

File: sources/platform/proxy/usage.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ If you need to test Apify Proxy before you subscribe, please [contact our suppor
 
 | Parameter           | Value / explanation |
 |---------------------|---------------------|
-| Hostname            | `proxy.apify.com`, alternatively you can use static IP addresses `18.208.102.16`, `35.171.134.41`. |
+| Hostname            | `proxy.apify.com`, alternatively you can use static IP addresses `18.208.102.16`, `35.171.134.41`. However, it is **strongly recommended to use the DNS name** (`proxy.apify.com`) as IP addresses are only available based on the current backend targets. |
```
Comment (fnesveda) [inline sources/platform/proxy/usage.md:35]: Is there any reason someone might want to connect directly through the IP addresses? I would be decisive and not list the direct connection option at all, and list only the hostname here, and then list the IP addresses in a separate paragraph outside the table. E.g. "the proxy runs on these two IP addresses, if you need to whitelist them in your firewall".

**Review by fnesveda — CHANGES_REQUESTED** [review]
Looks good, one note about the IP addresses 🙂

File: sources/platform/proxy/usage.md (hunk 1 above)
Comment (seyhello) [reply sources/platform/proxy/usage.md]: please, check last commit

**Review by fnesveda — APPROVED** [review]
Cool 👏

**Review by TC-MO — APPROVED** [review]
LGTM

**Pull request merged**